java/gast_to_code/gast_to_code_java.py

```diff
@@ -36,16 +36,42 @@ def handle_if(gast, lvl=0):
         return out
 
     def handle_none(gast):
-        pass
+        return "null"
 
     def handle_while(gast, lvl=0):
-        pass
+        test = router.gast_to_code(gast["test"], "java")
+
+        body_indent = "\n\t" + "\t" * lvl
+        closing_brace_indent = "\n" + "\t" * lvl
+        body = general_helpers.list_helper(gast["body"], "java", body_indent,
+                                           lvl + 1)
+
+        out = 'while (' + test + ') {' + body_indent + body + closing_brace_indent + "}"
+        return out
 
     def handle_for_range(gast, lvl=0):
-        pass
+        loop_init = router.gast_to_code(gast["init"], "java")
+        loop_test = router.gast_to_code(gast["test"], "java")
+        loop_update = router.gast_to_code(gast["update"], "java")
+
+        body_indent = "\n\t" + "\t" * lvl
```

Comment:
Consider creating helper functions for body_indent and closing_brace_indent since they are used in multiple places.